### ISSUE TITLE
AWS Backup Region Settings - ResourceTypeManagementPreference

### ISSUE THREAD

**Issue by noahtrilling** [hashicorp/terraform-provider-aws#22121]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS Backup now supports Advanced DynamoDB backup features including cross-account and cross-region copies. The AWS Backup region settings were updated to require opt-in to the advanced DynamoDB features. The aws_backup_region_settings resource should be updated to include the new ResourceTypeManagementPreference key referenced in the documentation links below.

### New or Affected Resource(s)

* aws_backup_region_settings

### Potential Terraform Configuration

```hcl
resource "aws_backup_region_settings" "test" {
  resource_type_management_preference = {
    "DynamoDB" = true
    "EFS" = true
  }
  resource_type_opt_in_preference = {
    "DynamoDB" = true
    "Aurora" = true
    "EBS" = true
    "EC2" = true
    "EFS" = true
    "FSx" = true
    "RDS" = true
    "Storage Gateway" = true
  }
}
```

### References

* https://docs.aws.amazon.com/aws-backup/latest/devguide/API_UpdateRegionSettings.html
* https://docs.aws.amazon.com/aws-backup/latest/devguide/API_DescribeRegionSettings.html
* https://docs.aws.amazon.com/aws-backup/latest/devguide/advanced-ddb-backup.html#advanced-ddb-backup-enable-cli

**Comment by ewbankkit** [comment]
@noahtrilling Thanks for raising this issue.
It has already been noticed in #22012. I'm going to close this one as a duplicate so that we can concentrate discussion in the linked issue.
Please add any additional comments there.

**Comment by ewbankkit** [comment]
The functionality will be released in **v3.69.0** of the Terraform AWS Provider.
